### PR TITLE
Update GNSS documentation to reflect new TCP and UDP connection availability

### PR DESCRIPTION
**Pull request by nirvn**
Text also reflects UI changes.

### CODE REVIEW

File: documentation/how-to/gnss.en.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ tx_slug: documentation_how-to_gnss
 
 QField can make use of the internal GNSS (Global Navigation Satellite
 System, like GPS, GLONASS, Galileo or Beidou). QField can also connect
-to external antennas through bluetooth and directly integrate their NMEA
-stream.
+to external antennas through NMEA streams over Bluetooth, TCP, or UDP
+connection.
```
Comment (m-kuhn) [inline documentation/how-to/gnss.en.md:11]: Shall we add a compatibility table OS/technology?

Comment (nirvn) [reply documentation/how-to/gnss.en.md:11]: Android: Bluetooth, TCP, UDP
iOS: TCP, UDP
Windows: TCP, UDP
Linux: Bluetooth, TCP, UDP

@m-kuhn , correct?

Comment (nirvn) [reply documentation/how-to/gnss.en.md:11]: @m-kuhn , done.

**Review by m-kuhn — APPROVED** [review]
Thanks

**Pull request merged**